libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1356,12 +1356,12 @@ void RCOutput::dshot_send(pwm_group &group, uint32_t time_out_us)
 #ifdef HAL_WITH_BIDIR_DSHOT
             // retrieve the last erpm values
             const uint16_t erpm = group.bdshot.erpm[i];
-
+#if HAL_WITH_ESC_TELEM
             // update the ESC telemetry data
             if (erpm < 0xFFFF && group.bdshot.enabled) {
                 update_rpm(chan, erpm * 200 / _bdshot.motor_poles, get_erpm_error_rate(chan));
```

Comment:
I think it would be better to add a null update_rpm to the dummy class, then the #if can be removed - I doubt it makes much difference to the size

Reply:
That one can be an empty function yes, but the other function requires the TelemetryData datastructure to be declared.. making the patch bigger.

Reply:
isn't the structure defined in the backend?